Docs/Practitioners/PdcPractitioner.md

```diff
@@ -0,0 +1,28 @@
+# PdcPractitioner
+
+Holds the board order and licensing information of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| fid  | string | Federation ID of the medical professional |
+| boardOrders | [BoardOrder](#boardOrder) array| A collection of the professional's board orders |
+| licenses | [License](#license) array | a collection of the professional's licenses |
+
+## Board Order
+
+Holds relevant information regarding a board order
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| category  | string | The type of board order issued |
+| actions | [CodedDescription](CodedDescription.md) array| The actions taken |
```

Comment:
You used "A collection of" earlier. Be consistent. "A collection of actions taken against a medical professional."